src/software/thunderscope/binary_context_managers.py

```diff
@@ -151,6 +151,9 @@ def setup_proto_unix_io(self, proto_unix_io):
         proto_unix_io.attach_unix_receiver(
             self.full_system_runtime_dir + "/log", RobotLog
         )
+        proto_unix_io.attach_unix_receiver(
+            self.full_system_runtime_dir + "/validation", ValidationProtoSet
+        )
```

Comment:
I'm not sure if I'm missing something, but when I played a replay I didn't get the validations visualized :confused: (though it was visualized when running it live)

Reply:
this only affects replays that were recorded from this branch, not replaying old replays

Reply:
I tried running goalie_tactic_test on this branch and it didn't show the validations